Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -146,18 +147,37 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   }
 }
 
-// FIXME: This macro is not safe for nil arguments as types in this case cannot be inferred.
+#define ARRAY_FROM_ARGS(...) ({\
+ [NSArray arrayWithObjects: !(sizeof( (char[]){#__VA_ARGS__} ) == 1) ? __VA_ARGS__ : [NSNull null], nil];\
+})
+
+#define MS_DISPATCH_SELECTOR_OBJECT(result, class, selectorName, ...) ({ \
+ NSArray* array = ARRAY_FROM_ARGS(__VA_ARGS__);\
+ id instance = class;\
```

Comment:
Please avoid built-in words usage

Reply:
Updated, tried to remove as much as possible